curated_transformers/util/serde.py

```diff
@@ -33,16 +91,23 @@ def load_model_from_checkpoints(
         PyTorch module into which the parameters are to be loaded.
     :param filepaths:
         Paths to PyTorch checkpoints.
+    :param checkpoint_type:
+        Type of checkpoint being loaded.
     :param state_dict_converter:
         Callback to convert Hugging Face state dicts to the
-        `curated-transformers` format.
+        ``curated-transformers`` format.
     :param tensor_to_param_converter:
         Callback to perform custom conversions of the loaded parameters.
         Useful for loading quantized weights.
     :param device:
         Device in which to place the loaded parameters.
     """
-    state_dicts = _load_state_dicts_from_checkpoints(filepaths)
+    checkpoint_type_to_loader = {
```

Comment:
I think it's cleaner if we add this as a method (even if private) to the enum.

Reply:
Moved to the enum.